omnisafe/utils/exp_grid_tools.py

```diff
@@ -49,3 +54,45 @@ def valid_str(vals: list | str) -> str:
     str_v = str(vals).lower()
     valid_chars = f'-_{string.ascii_letters}{string.digits}'
     return ''.join(c if c in valid_chars else '-' for c in str_v)
+
+
+def train(
+    exp_id: str,
+    algo: str,
+    env_id: str,
+    custom_cfgs: NamedTuple,
```

Comment:
```suggestion
from omnisafe.typing import Config
custom_cfgs: Config
```
I will fix it in my ``mypy`` PR #221. I just mark it but do not request change.

Reply:
got it